src/query/expression/src/block.rs

```diff
@@ -122,6 +124,29 @@ impl DataBlock {
         }
     }
 
+    fn check_columns_valid(columns: &Vec<BlockEntry>, num_rows: usize) -> Result<()> {
```

Comment:
```suggestion
    #[cfg(debug_assertions)]
    fn check_columns_valid(columns: &Vec<BlockEntry>, num_rows: usize) -> Result<()> {
```

Reply:
this function is used during serializing, it's enabled not only for debug.